pages/dashboard/index.tsx

```diff
@@ -1,60 +1,25 @@
-import type {NextComponentType, NextPage} from 'next'
-import {
-    Avatar,
-    Box,
-    Button,
-    Menu,
-    MenuItem,
-    Toolbar,
-} from "@mui/material";
-import React from "react";
+import React, {ReactElement} from "react";
+import DashboardLayout from "../../components/layouts/dashboard/dashboardLayout/DashboardLayout";
+import {Typography} from "@mui/material";
+import {PageWithLayout} from "../_app";
 
-import {usePopupState, bindTrigger, bindMenu} from 'material-ui-popup-state/hooks'
-import DBDrawerLayout from "../../components/layouts/dashboard/dbDrawer/dbDrawerLayout/DBDrawerLayout";
 
+const DashboardHome: PageWithLayout = () => {
 
-const DashboardHome: NextPage = () => {
+    return <></>;
 
-    return (
-        <>
-            <DashboardAppBar/>
-
-            <Box sx={{ marginTop: 8,
-                display: 'flex',
-                flexDirection: 'column',
-                alignItems: 'center'}}>
-
-            </Box>
-        </>
-    )}
-
-const DashboardAppBar: NextComponentType = () => {
+}
 
-    const AccountMenuPopupState = usePopupState({ variant: 'popover', popupId: 'accountMenu' })
+DashboardHome.getLayout = function getLayout(page: ReactElement) {
 
     return (
-        <>
-            <DBDrawerLayout/>
-
-            <Toolbar sx={{width:"90%", ml:"10%"}}>
-
-                <Box sx={{width:"100%"}}></Box>
-                <Button
-                    {...bindTrigger(AccountMenuPopupState)}
-                    variant={"text"}
-                    sx={{m:1}}>
-                    <Avatar/>
-                </Button>
-                <Menu {...bindMenu(AccountMenuPopupState)}>
-
-                    <MenuItem onClick={AccountMenuPopupState.close}>Cake</MenuItem>
-                    <MenuItem onClick={AccountMenuPopupState.close}>Death</MenuItem>
-                </Menu>
-
-            </Toolbar>
+        <DashboardLayout>
+            <Typography variant={"h2"} sx={{m:10}}>Dashboard Home</Typography>
+            {page}
```

Comment:
anything that is not the dashboard layout and page id move to the return function this might help with rerender same with the rest of the dashboard pages 
```suggestion
            {page}
```